openmetadata-service/src/main/java/org/openmetadata/service/events/ChangeEventHandler.java

```diff
@@ -84,32 +77,19 @@ public Void process(
           changeEvent.setEntity(JsonUtils.pojoToMaskedJson(entity));
         }
 
-        changeEventDAO.insert(JsonUtils.pojoToJson(changeEvent));
+        // Thread are created in FeedRepository Directly
+        Entity.getCollectionDAO().changeEventDAO().insert(JsonUtils.pojoToJson(changeEvent));
 
-        // Add a new thread (change event itself should not be for the thread) to the entity for
-        // every change event
-        // for the event to appear in activity feeds
-        if (!changeEvent.getEntityType().equals(Entity.THREAD)
-            && (AlertUtil.shouldProcessActivityFeedRequest(changeEvent))) {
-          for (Thread thread :
-              listOrEmpty(
-                  FeedUtils.getThreadWithMessage(
-                      feedMessageDecorator, changeEvent, loggedInUserName))) {
-            // Don't create a thread if there is no message
-            if (thread.getMessage() != null && !thread.getMessage().isEmpty()) {
-              feedRepository.create(thread, changeEvent);
-              String jsonThread = mapper.writeValueAsString(thread);
-              WebSocketManager.getInstance()
-                  .broadCastMessageToAll(WebSocketManager.FEED_BROADCAST_CHANNEL, jsonThread);
-              if (changeEvent.getEventType().equals(ENTITY_DELETED)) {
-                deleteAllConversationsRelatedToEntity(getEntity(changeEvent), collectionDAO);
-              }
-            }
-          }
+        // Delete all conversations related to the entity
+        if (changeEvent.getEventType().equals(ENTITY_DELETED)) {
+          deleteAllConversationsRelatedToEntity(getEntity(changeEvent), Entity.getCollectionDAO());
```

Comment:
this we should move to delete operation in EntityRepository. We don't want to do this heavy operation on a callback